lib/policy_machine.rb

```diff
@@ -267,6 +267,16 @@ def accessible_objects(user_or_attribute, operation, options = {})
     end
   end
 
+  def accessible_ancestor_objects(user_or_attribute, operation, root_object, options = {})
+    if policy_machine_storage_adapter.respond_to?(:accessible_ancestor_objects)
+      policy_machine_storage_adapter.accessible_ancestor_objects(user_or_attribute, operation, root_object, options)
+    else
+      Warn.once "WARNING: accessible_ancestor_objects is not implemented for storage adapter " \
+                "#{policy_machine_storage_adapter.class}. Returning all accessible_objects."
+      accessible_objects(user_or_attribute, operation, options)
```

Comment:
Should returning all accessible objects be the default behavior if the method isn't implemented, or should it raise? 

I guess my concern is that someone could use this method and work on the return assuming that it is scoped to _only_ ancestor objects, which could lead to Bad Things if they don't also verify that each returned item is an ancestor.

Reply:
The warning should address that, but there is an argument to be made that it should return `[]` or error out. @mdsol/team04 any opinions?

Reply:
Having it default to `accessible_objects` is disingenuous, and I relying on people to read errors is...not reliable. I could go either way on this, but I think we should just let it raise if it's not implemented.